docs/CHANGELOG.asciidoc

```diff
@@ -49,6 +49,34 @@ Review important information about the {kib} 8.x releases.
 
 Review the following information about the {kib} 8.8.2 release.
 
+[float]
+[[known-issues-8.8.2]]
+=== Known issues
+
+// tag::known-issue-161249[]
+[discrete]
+.Kibana can run out of memory during an upgrade when there are many {fleet} agent policies.
+[%collapsible]
+====
+*Details* +
+Due to a schema version update, during {fleet} setup in 8.8.x, all agent policies are being queried and deployed. 
+This action triggers a lot of queries to the Elastic Package Registry (EPR) to fetch integration packages.  As a result,
+there is an increase in Kibana's resident memory usage (RSS). 
+
+*Impact* +
+Because the default batch size of `100` for schema version upgrade of {fleet} agent policies is too high, this can 
+cause Kibana to run out of memory during an upgrade.  For example, we have observed 1GB Kibana instances run
+out of memory during an upgrade when there were 20 agent policies with 5 integrations in each. 
+
+*Workaround* +
+There are two workaround options available:
+- Increase the Kibana instance size to 2GB. So far, we are not able to reproduce the issue with 2GB instances.
+- Set `xpack.fleet.setup.agentPolicySchemaUpgradeBatchSize` to `2` in the `kibana.yml` and restart the Kibana instance(s).
+In 8.9.0, we are addressing this by changing the default batch size to `2`.
```

Comment:
```suggestion
Two workaround options are available:

* Increase the Kibana instance size to 2GB. So far, we are not able to reproduce the issue with 2GB instances.
* Set `xpack.fleet.setup.agentPolicySchemaUpgradeBatchSize` to `2` in the `kibana.yml` and restart the Kibana instance(s).

In 8.9.0, we are addressing this by changing the default batch size to `2`.
```
Just fixing the formatting so that the two options display as a bulleted list.